src/guide/6-agent-birth-death/3-agent-death.rst

```diff
@@ -1,2 +1,56 @@
 Agent Death
 ===========
```

Comment:
Minor grumble, this is something I do dislike. The need to always return something (probably ALIVE) in non dying functions to avoid compiler warnings. Agent could fix this with templating, but makes the FLAMEGPU_AGENT_FUNCTION require even more faff.